target/linux/ath79/dts/qca9563_tplink_archer-c7-v5.dts

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9563_tplink_archer-x7-v5.dtsi"
+
+/ {
+	compatible = "tplink,archer-c7-v5", "qca,qca9563";
+	model = "TP-Link Archer C7 v5";
+};
+
+&mtdparts {
+	factory-uboot@0 {
```

Comment:
Name of DT node should be partiton@xxxx not <partiton-name>@xxxx.

It is also wrong for the Archer A7 v5, where is suppose this partition layout is partly originated from.